### ISSUE TITLE
sel_latlon with metric and nearest

### ISSUE THREAD

**Issue by vergauwenthomas** [vergauwenthomas/xarray_for_latlon#2]
Implement the method `sel_latlon`, which is a wrapper on the `xr.sel`.  It allows a user to use the sel method on non-index coordinates (n-D).
In addition a metric (see sklearn.metrics.pairwise) is passed as argument. 

This method must first find the nearest point index location, and then apply the `.sel` on that point.
